tests/unit/daysBetween-spec.js

```diff
@@ -0,0 +1,31 @@
+const daysBetween = require('../../src/util/daysBetween')
+const { expect } = require('chai')
+
+const testCases = [{
+  startDate: '2018-03-01',
+  endDate: '2018-03-31',
+  expected: 31
+}, {
+  startDate: '2018-02-01',
+  endDate: '2018-02-28',
+  expected: 28
+}, {
+  startDate: '2020-03-01',
+  endDate: '2020-03-31',
+  expected: 31
+}, {
+  startDate: '2020-02-01',
+  endDate: '2020-02-29',
+  expected: 29
+}]
```

Comment:
These tests are non-exhaustive, but should provide some protection against future changes.